inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py

```diff
@@ -1,173 +1,141 @@
-#!/usr/bin/env python
-"""
- Copyright (C) 2018-2021 Intel Corporation
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
-"""
-from __future__ import print_function
+#!/usr/bin/env python3
```

Comment:
Will it work on Windows where python3 is absent?

Reply:
I think it will take python 3 launcher https://docs.python.org/3/using/windows.html#customizing-default-python-versions 
@dpigasin  double check it